charts/ocis/templates/web/deployment.yaml

```diff
@@ -56,9 +56,10 @@ spec:
             {{- else }}
               value: {{ .Values.features.externalUserManagement.oidc.issuerURI | quote }}
             {{- end }}
-
             - name: WEB_OIDC_CLIENT_ID
               value: {{ .Values.features.externalUserManagement.oidc.webClientID | quote }}
+            - name: WEB_OIDC_SCOPE
+              value: {{ .Values.features.externalUserManagement.oidc.WebClientScope | quote }}
```

Comment:
I'm not sure if we should add those web related config options in this generic section, because it might be a better fit in the `services.web` section.

I'm open for discussions, but I see it becoming bloated if we expose more Web only settings in this section

Reply:
Probably it was already a bad decision to not put `.Values.features.externalUserManagement.oidc.webClientID` in the web service config part

Reply:
From discussion, we should structure it like this:

```
services:
  web:
    config:
      oidc:
        # URI where to redirect the user after a logout was performed. Defaults to the URI of the login page.
        postLogoutRedirectURI: ""
        # Specifies the target url valid for the logged out / access denied page.
        loginURL:
        WEB_OIDC_CLIENT_ID // moved from externalUserManagement with proper naming
        WEB_OIDC_SCOPE // newly added as in this PR "webClientScope"
```